metrics/metrics.go

```diff
@@ -34,14 +43,6 @@ var (
 			Help:      "Counter of snapshots.",
 		})
 
-	TxnCmdCounter = prometheus.NewCounterVec(
```

Comment:
Is it not useful anymore?

Reply:
Command duration already contains count so it can be removed.